tools/bug-report/pkg/bugreport/flags.go

```diff
@@ -96,10 +96,9 @@ func addFlags(cmd *cobra.Command, args *config2.BugReportConfig) {
 	cmd.PersistentFlags().StringVar(&outputDir, "output-dir", "",
 		"Set a specific directory for output archive file.")
 
-	// requests per second limit
-	cmd.PersistentFlags().IntVar(&args.RequestsPerSecondLimit, "rps-limit", 0,
```

Comment:
I don't think we should remove flags directly like this without any deprecation notes. Can we provide an alternative option instead?

Reply:
My reasoning for removing the rate is that it doesn't make much sense to follow any deprecation procedure the bug-report utility. Isn't this essentially a script that's run only in certain circumstances?

I'm happy to add it back if there are concerns around compatibility, though.

Reply:
Sounds good.